app/src/main/java/com/summer/practice/tvtracker/LoginActivity.kt

```diff
@@ -0,0 +1,37 @@
+package com.summer.practice.tvtracker
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Button
+import android.widget.EditText
+import android.widget.TextView
+
+class LoginActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_login)
+        val emailField = findViewById<EditText>(R.id.emailField)
+        val passwordField = findViewById<EditText>(R.id.passwordField)
+        val errorMessage = findViewById<TextView>(R.id.errorMessage)
+        val loginButton = findViewById<Button>(R.id.LoginButton)
+        val linkToRegistration = findViewById<TextView>(R.id.linkToRegistration)
```

Comment:
let's start to add viewBinding into the project and replace the whole "get view references" with the viewBinding magic

Check this out: 
https://developer.android.com/topic/libraries/view-binding